custom/crediblex/portfolio/loc/src/main/java/com/crediblex/fineract/portfolio/loc/service/LineOfCreditDataValidator.java

```diff
@@ -120,11 +124,26 @@ public void validateForUpdate(String json) {
             "authorizedSignatoryName", "authorizedSignatoryPhone", "authorizedSignatoryEmail", "va",
             "distributionPartner", "bankTransferFee", "specialConditions", "latePaymentFee"
         ));
-        this.fromApiJsonHelper.checkForUnsupportedParameters(typeOfMap, json, supportedParameters);
+        
+        try {
+            // Create a filtered JSON without clientId for the unsupported parameters check
+            String filteredJson = json;
+            if (json.contains("\"clientId\":null")) {
+                filteredJson = json.replace("\"clientId\":null,", "").replace(",\"clientId\":null", "");
+            }
+            
+            this.fromApiJsonHelper.checkForUnsupportedParameters(typeOfMap, filteredJson, supportedParameters);
+        } catch (Exception e) {
+            throw e;
+        }
 
         if (this.fromApiJsonHelper.parameterExists("name", element)) {
-            final String name = this.fromApiJsonHelper.extractStringNamed("name", element);
-            baseDataValidator.reset().parameter("name").value(name).notBlank().notExceedingLengthOf(100);
+            try {
+                final String name = this.fromApiJsonHelper.extractStringNamed("name", element);
+                baseDataValidator.reset().parameter("name").value(name).notBlank().notExceedingLengthOf(100);
```

Comment:
No need to catch this exception and throw it again. Its also not a specific exception.